scripts/postinstall.js

```diff
@@ -0,0 +1,9 @@
+// There doesn't appear to be a way to skip `postinstall`,
+// as doing `yarn --ignore-scripts` could prevent native packages from building:
+// https://github.com/yarnpkg/yarn/issues/4100
```

Comment:
There is another way to achieve conditionally running a script mentioned in this thread, however I don't think it's cross-platform, so I chose a simple npm script instead.